cpython/patches/0007-Remove-trampoline-definitions.patch

```diff
@@ -0,0 +1,88 @@
+From dfb8c93d4e9754e56ce805088e33fed2668c2834 Mon Sep 17 00:00:00 2001
+From: Hood Chatham <roberthoodchatham@gmail.com>
+Date: Mon, 19 Jun 2023 12:32:05 -0700
+Subject: [PATCH 7/7] Remove trampoline definitions
+
+We need to make our own JSPI aware versions
```

Comment:
Could you please add a comment on where we locate our own version of these methods, for those who see this file later?